primitives/runtime/src/lib.rs

```diff
@@ -622,8 +622,13 @@ macro_rules! impl_outer_config {
 		}
 
 		$crate::paste::item! {
+			#[cfg(any(feature = "std", test))]
+			#[doc(hidden)]
+			use $crate::serde as _sp_runtime_hidden_serde;
```

Comment:
This enforce only one call to impl_outer_config :-/ if this is considered a blocking point, I can try to work around by having construct_runtime giving the path to serde to impl_outer_config.

Reply:
Fixed